package/Dockerfile

```
@@ -235,61 +299,68 @@ ENV ETCD_UNSUPPORTED_ARCH ${ETCD_UNSUPPORTED_ARCH}
 
 
 # Enable exporting of the k3s images as part of the build process.
-FROM --platform=$BUILDPLATFORM registry.suse.com/bci/golang:1.24 AS images
-
-ENV GOMODCACHE=/root/.cache/go/modcache
-ENV GOCACHE=/root/.cache/go/cache
-
+FROM --platform=$BUILDPLATFORM go-builder AS images
+WORKDIR /src
+ENV CGO_ENABLED=1
```

Shouldn't CGO be disabled here?
```suggestion
ENV CGO_ENABLED=0
```

---

CGO is enabled only for the airgap binary (export-images). It was enabled before and it still is, so that remain unchanged.

I did try previously to turn it off but it failed. That stage downloads some btrfs-devel and gpgme-devel libraries and I'm guess we're making use of those things via CGO.